### DISCUSSION TITLE
Control Panel sidebar not sticky

### DISCUSSION THREAD

**Discussion by mladen-dordevic** [ExpressionEngine/ExpressionEngine#2950] [Feature Requests]
Please consider making sidebar in Control Panel scroll independently of the content of the main window as when the main windows has a scroll content it scrolls sidebar as well and prevents a user from making a quick pick of nav items.

EE 6.0.0-b.3

**Comment by Colbitron** [comment]
Edit: my bad. I'm rereading this and it's clear to me now this is a bit different than my issue. Apologies.

**Comment by chunkitdesign** [comment]
I second this.
